index.html

```diff
@@ -11,22 +11,34 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-  <header class="Toolbar">
-    <div class="logo">
-      <a href="index.html" class="Mylogo">JIC Software</a>
-    </div>
-    <div class="menu-pc">
-      <a class="item-menu-pc">Portfolio</a>
-      <a class="item-menu-pc">About</a>
-      <a class="item-menu-pc-group">
-      <a class="item-menu-pc">Contact</a>
-        <img class="item-menu-pc-icon" src="images/mail_icon.png" alt="email icon">
-      </a>
+  <header>
+    <div class="Toolbar">
+      <div class="logo">
+        <a href="#" class="Mylogo">JIC Software</a>
+      </div>
+      <div class="menu-pc">
+        <a href="#portfolio" class="item-menu">Portfolio</a>
+        <a href="#about" class="item-menu">About</a>
+        <div href="#contact" class="item-menu-group">
+          <a href="#contact" class="item-menu">Contact</a>
+          <img href="#contact" class="item-menu-icon" src="images/mail_icon.png" alt="email icon">
+        </div>
+      </div>
+      <nav id="burguer-menu" class="burguer-menu">
+          <img class="burguer-menu-icon" src="images/burguer_menu_icon.png" alt="burguer menu icon">
+      </nav>
     </div>
-    <nav class="menu-mobile">
-        <img class="burguer-menu" src="images/burguer_menu_icon.png" alt="burguer menu icon">
-    </nav>
+
+    <div class="menu-bar-movil">
+      <nav class="slidemenu" id="slidemenu">
+        <a class="btn-close-slide-menu">X</a>
+        <a href="#portfolio" class="item-menu-mobile">Portfolio</a>
+        <a href="#about" class="item-menu-mobile">About</a>
+        <a href="#contact" class="item-menu-mobile">Contact</a>
+      </nav>
+		</div>  <!-- menu movil-->    
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please stick to the Figma design your design is still not the same as the Figma design check below :point_down:  the comparison between your and Figma design :slightly_smiling_face: 

| Your Design   |      Figma design      | 
|---------- |:----------------:|
| ![Screenshot from 2023-04-18 20-09-38](https://user-images.githubusercontent.com/45226848/232822225-26c2a71e-a6d4-4722-a86f-501eec3aedde.png) | ![Figma-design](https://user-images.githubusercontent.com/62290545/232596283-5b2c7c38-785c-402c-9ee2-b09ac8f509a0.png)  | 

As mentioned by the previous reviewer :point_up_2: follow the following hints :page_facing_up: 

 - Adjusting the positioning of the nav menu elements and the close button to match the design.
 - Changing the colour of the close button to match the Figma template.
 - Changing the close button size to match the design.

Reply:
Thankyou @aamirkhan2478 . There was some error in my github because I made all this corrections before re-asking a new code review.
I re-ask again after verify what happen with my last commit, which as i told you , it has all the corrections